eng/common/pipelines/templates/steps/verify-changelog.yml

```diff
@@ -12,13 +12,13 @@ parameters:
 steps:
   - task: Powershell@2
     inputs:
-      filePath: /eng/common/scripts/Verify-ChangeLog.ps1
+      filePath: $(Build.SourcesDirectory)/eng/common/scripts/Verify-ChangeLog.ps1
       arguments: >
         -PackageName ${{ parameters.PackageName }}
         -ServiceName ${{ parameters.ServiceName }}
         -RepoRoot $(Build.SourcesDirectory)
         -RepoName $(Build.Repository.Name)
-        -ForRelease ${{ parameters.ForRelease }}
+        -ForRelease $${{ parameters.ForRelease }}
```

Comment:
Looks like you added an extra `$` which is not correct.

Reply:
Talked offline with @chidozieononiwu and this is correct as we need the `$` in front of the true or false value for passing to powershell.